docs/local.md

```diff
@@ -64,25 +73,44 @@ on port `:1053`.
 ```shell script
 dig @localhost -p 1053 roundrobin.cloud.example.com +short
 ```
-Should return ***two A records*** from both clusters:
+Should return ***two A records*** from both clusters (IP addresses and order may differ):
+```
+172.20.0.2
+172.20.0.5
+172.20.0.4
+172.20.0.6
+```
+
+You can verify that correct IP addresses of all the nodes in both clusters were populated:
+```shell script
```

Comment:
@jkremser I know that it might not be the scope of this PR, but `shell script` is not a valid shell identifier, so it doesn't have an effect on rendering.
it should be either set to `shell-script` or any other [valid alias](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5602-L5605) for GH [syntax highlighting](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting) (I like `sh` for shortness :-) ) or not set at all.

Reply:
good finding, amended and changed also on other places